test/GreaterThanOrEqualsToValidator.test.js

```diff
@@ -1,8 +1,10 @@
 const Validator = require('../src/GreaterThanOrEqualsToValidator');
 
 describe ('GreaterThanOrEqualsToValidator', () => {
+    const createDate = () => new Date(1500000000000);
+
     context ('#constructor', () => {
-        it ('should throw an Error if the argument is not a number', () => {
+        it ('should throw an Error if the argument is not a number nor Date', () => {
```

Comment:
`should throw an Error if the argument is not a number nor a Date instance`

Reply:
Listo.